docs/installation_guide.rst

```diff
@@ -313,6 +313,7 @@ After the app is built, if everything went well you should see the below text in
 
 If you get any errors about missing dependencies, try to install them via npm and run again; consult the troubleshooting section for other errors.
 
+3. Alternatively, you can also choose to build the application. In order to do so, navigate to your rotki development directory and execute the ``package.bat`` file. NOTE: You will need to edit the directory in Line 30 to the name of your rotki development directory.
```

Comment:
What are you referring to here? 

> NOTE: You will need to edit the directory in Line 30 to the name of your rotki development directory.

Lines may change also so just pointing to a line won't work here.

Reply:
``
Rem also for Windows we expect a pysqlcipher3 project to exist to properly
Rem build and install that into the venv since pip install pysqlcipher3 without
Rem having manually compiled sqlcipher will not work.
Rem Reference: https://rotki.readthedocs.io/en/latest/installation_guide.html#sqlcipher-and-pysqlcipher3
cd ../pysqlcipher3
python setup.py build
python setup.py install
cd ../rotkehlchen
``

In this section of the script the pysqlcipher3 and rotkehlchen directories may be named differently (in theory pysqlcipher3 shouldn't, but if you end up going the route of manual installation it may be). Rotkehlchen is whatever dir you locally cloned the repo / downloaded the source to. Noted to make it clearer.

Reply:
If you followed the default guide pysqlcipher3 should be the same. With the new name of rotki the directory may indeed be different though so it's a good note to have.

But instead of pointing to a line number which will be out of date soon can you instead link to the code in a particular commit so that the person who reads it can just click the link and go to a non-modifiable version of the code?

Reply:
That's a good idea; will do that.